packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -0,0 +1,50 @@
+import { IForwarder, IPrismInput } from '@stoplight/prism-core';
+import { IHttpOperation, IServer } from '@stoplight/types';
+import axios from 'axios';
+import { IHttpConfig, IHttpRequest, IHttpResponse } from '../types';
+
+export class HttpForwarder
+  implements IForwarder<IHttpOperation, IHttpRequest, IHttpConfig, IHttpResponse> {
+  public async forward(opts: {
+    resource?: IHttpOperation;
+    // todo: IHttpRequest should be enough
+    input: IPrismInput<IHttpRequest>;
+  }): Promise<IHttpResponse> {
+    if (!opts.resource) {
+      throw new Error('Missing spec');
+    }
+    if (!opts.resource.servers || opts.resource.servers.length === 0) {
+      throw new Error('Server list is missing in spec');
+    }
+
+    const response = await axios({
+      method: opts.input.data.method,
+      url: this.resolveServerUrl(opts.resource.servers[0]) + opts.input.data.url.path,
+      params: opts.input.data.url.query,
+      responseType: 'text',
+      data: opts.input.data.body,
+      headers: opts.input.data.headers,
+    });
+
+    return {
+      statusCode: response.status,
+      headers: response.headers,
+      body: response.data,
+    };
+  }
+
+  private resolveServerUrl(server: IServer) {
```

Comment:
@StefanDywersant 
Curious: why not using the serverMatch function from the router module?

Reply:
resolveServerUrl picks first server from list and eventually fills it with default values for variables. matchServer from router package finds server for given baseUrl. So The purpose of those methods is different -> I can't use matchServer.